### PR TITLE
Improve DNS api documentation

### PR DESCRIPTION
**Pull request by galpeter**
IoT.js-DCO-1.0-Signed-off-by: Peter Gal pgal.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by galpeter** [comment]
Related issue: #873

File: docs/api/IoT.js-API-DNS.md [hunk 1]
```diff
@@ -6,61 +6,78 @@ The following shows dns module APIs available for each platform.
...
-* `hostname <String>`
-* `options <Object> | <Number>`
-  * `family <Number>` - The record family. Must be 4 or 6. IPv4 and IPv6 addresses are both returned by default.
-  * `hints <Number>` - One or more supported getaddrinfo flags. Multiple flags may be passed by bitwise ORing their values.
-* `callback <Function(err: Error, address: String, family: Number)>`
-  * `err <Error>`
-  * `address <String>` - A string representation of an IPv4 or IPv6 address.
-  * `family <Number>` - 4 or 6, denoting the family of address.
+* `hostname {String}` Hostname to be resolved.
+* `options {Object> | <Number>}`
```
Comment (daeyeon) [inline docs/api/IoT.js-API-DNS.md]: I understand that it's hard to find bracket missing mistakes like this. I have no idea to validate this. Maybe we could use `https://github.com/nodejs/doc-tool` and check the result.

File: docs/api/IoT.js-API-DNS.md [hunk 2]
```diff
@@ -6,61 +6,78 @@ The following shows dns module APIs available for each platform.
...
 ### `dns.ADDRCONFIG`
-* `<Number>`
+* `{Number}`
 
 Returned address types are determined by the types of addresses supported by the current system.
 
 
 ### `dns.V4MAPPED`
-* `<Number>`
+* `{Number}`
```
Comment (daeyeon) [inline docs/api/IoT.js-API-DNS.md]: {Number} -> {number} for all the same

File: docs/api/IoT.js-API-DNS.md [hunk 3]
```diff
@@ -6,61 +6,78 @@ The following shows dns module APIs available for each platform.
...
 ### `dns.lookup(hostname[, options], callback)`
-* `hostname <String>`
-* `options <Object> | <Number>`
-  * `family <Number>` - The record family. Must be 4 or 6. IPv4 and IPv6 addresses are both returned by default.
-  * `hints <Number>` - One or more supported getaddrinfo flags. Multiple flags may be passed by bitwise ORing their values.
-* `callback <Function(err: Error, address: String, family: Number)>`
-  * `err <Error>`
-  * `address <String>` - A string representation of an IPv4 or IPv6 address.
-  * `family <Number>` - 4 or 6, denoting the family of address.
+* `hostname {String}` Hostname to be resolved.
```
Comment (daeyeon) [inline docs/api/IoT.js-API-DNS.md]: {String}  -> {string}

File: docs/api/IoT.js-API-DNS.md [hunk 4]
```diff
@@ -6,61 +6,78 @@ The following shows dns module APIs available for each platform.
...
-        - [`dns.ADDRCONFIG`](#dnsaddrconfig)
-        - [`dns.V4MAPPED`](#dnsv4mapped)
-    - [Module Functions](#module-functions)
-        - [dns.lookup(hostname[, options], callback)](#dnslookuphostname-options-callback)
+※ dns.lookup currently only returns IPv4 addresses. Support for IPv6 addresses are on the roadmap.
 
 
 # DNS
 
+The `dns` module provides a method to perform host name resolution. The functionallity is
```
Comment (bzsolt) [inline docs/api/IoT.js-API-DNS.md]: functionality

**Comment by galpeter** [comment]
@daeyeon I've updated the PR

File: docs/api/IoT.js-API-DNS.md [hunk 5]
```diff
@@ -6,61 +6,78 @@ The following shows dns module APIs available for each platform.
...
 
 ```js
-const dns = require('dns');
+var dns = require('dns');
 dns.lookup ('localhost', 4, function(err, ip, family) {
-  assert.equal (err, null);
-  assert.equal (isIPv4 (ip), true);
-  assert.equal (ip, '127.0.0.1');
-  assert.strictEqual (family, 4);
+  console.log("ip: " + ip + " family: " + family);
```
Comment (daeyeon) [inline docs/api/IoT.js-API-DNS.md]: Can you use single quotes for all the samples? Others LGTM.

**Comment by galpeter** [comment]
@daeyeon I've fixed the quotes as requested and updated the PR.

**Pull request merged**